### PR TITLE
A82 update: fix field name and improve wording

### PR DESCRIPTION
**Pull request by markdroth**

### CODE REVIEW

**Review by ejona86 — APPROVED** [review]

File: A82-xds-system-root-certs.md [hunk 1]
```diff
@@ -66,7 +66,7 @@ to use system root certs, it will configure the TlsCreds code to do that.
 
 ### Temporary environment variable protection
 
-Use of the `use_system_root_certs` field in CDS and LDS will be guarded
+Use of the `system_root_certs` field in CDS and LDS will be guarded
```
Comment (ejona86) [inline A82-xds-system-root-certs.md:69]: Remove "LDS" from here as well?

Comment (markdroth) [reply A82-xds-system-root-certs.md]: Done.

Also improved the wording you pointed out about "ignoring" the field in LDS.  PTAL.

File: A82-xds-system-root-certs.md [hunk 2]
```diff
@@ -66,9 +67,9 @@ to use system root certs, it will configure the TlsCreds code to do that.
 
 ### Temporary environment variable protection
 
-Use of the `use_system_root_certs` field in CDS and LDS will be guarded
-by the `GRPC_EXPERIMENTAL_XDS_SYSTEM_ROOT_CERTS` env var.  The env var
-guard will be removed once the feature passes interop tests.
+Use of the `system_root_certs` field in CDS will be guarded by the
```
Comment (ejona86) [inline A82-xds-system-root-certs.md:70]: Doh. Now you need the LDS language here again, because we're changing the validation logic for LDS.

Comment (markdroth) [reply A82-xds-system-root-certs.md]: Lol, done.

**Review by ejona86 — APPROVED** [review]

**Pull request merged**